src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/casedocuments/DocumentHandlerBase.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.casedocuments;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.util.StringUtils;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.ContestedUploadedDocumentData;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class DocumentHandlerBase {
+
+    private final ObjectMapper objectMapper;
+
+    public DocumentHandlerBase(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    protected List<ContestedUploadedDocumentData> getDocumentCollection(Map<String, Object> caseData, String collection) {
+        if (StringUtils.isEmpty(caseData.get(collection))) {
```

Comment:
Can we extract this to a Util class if we already don't have this? I think apache commons have something like this but returns an immutable empty list, if I recall it correctly.